java/java.lsp.server/vscode/package.json

```diff
@@ -35,7 +35,12 @@
 					],
 					"default": null,
 					"description": "Specifies the JDK on which the Java language server should be run"
-				}
+				},
+                                "netbeans.verboseConsole": {
+                                        "type": "boolean",
+                                        "default": false,
+                                        "description": "Enables verbose messages from the Java Lanuage server on the console"
```

Comment:
Re. "on the console" - I don't think this makes sense from the point of view of VSCode user. The user sees no "console" as far as I can say. "Enables debug messages from the Java Language Server" (in the output tab).

Btw. typo "Lanuage".